packages/xo-web/src/xo-app/settings/remotes/index.js

```diff
@@ -49,12 +49,12 @@ const COLUMN_STATE = {
   itemRenderer: remote => (
     <div>
       <StateButton
-        disabledLabel={_('remoteDisconnected')}
+        disabledLabel={_('remoteDisabled')}
         disabledHandler={enableRemote}
-        disabledTooltip={_('remoteConnectTip')}
-        enabledLabel={_('remoteConnected')}
+        disabledTooltip={_('enableRemote')}
+        enabledLabel={_('remoteEnabled')}
         enabledHandler={disableRemote}
-        enabledTooltip={_('remoteDisconnectTip')}
```

Comment:
Are these messages still used?

Reply:
It will be used when exposing the connection state.

Reply:
I think it would be best to simply remove them for the time being.